src/domain/events/eventsTable/eventsTableRow/EventsTableRow.tsx

```diff
@@ -134,7 +134,11 @@ const EventTableRow: React.FC<Props> = ({
           </div>
         </td>
         <td className={styles.publisherColumn}>
-          <OrganizationName id={publisher} />
+          {publisher !== 'others' ? (
```

Comment:
I think you could actually move this logic to `OrganizationName` component and wrap the name in `<span>` if the publisher is "others" 🤔 It also already uses query for organizations so it will also translate the publisher name, no need for static "External" 🙂

Reply:
If we just blindly move this logic to OrganizationName component it would then add the highlight class to all places where the component is used where as we only want to highlight them on the EventTableRow. But yeah, I guess there could be neater way of doing this 🤔

Reply:
> It would then add the highlight class to all places where the component is used

True, I don't think it is that big of an issue.. 🤔 Alternatively could just wrap that OrganizationName with the span here so that it don't affect anywhere else 🙂

Reply:
OR could add some new true/false property to OrganizationName which would only enable styling here 😁

Reply:
I feel like the cleanest way to implement this is the span wrap around OrganizationName as it doesn't complicate the underlying components unnecessarily.